packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -328,4 +328,26 @@ export default class ComponentsTableRoute extends Route {
       </Hds::Table::Row>
     </:body>
   </Hds::Table>
+  {{! <h4 class="dummy-h4">
+    Static table with custom column widths
+  </h4> }}
+  <h4 class="dummy-h4">
+    Sortable table (all columns sortable)
+  </h4>
+  <Hds::Table::TableSortable
+    @model={{this.model}}
+    @sortBy={{this.sortBy}}
+    @sortOrder={{this.sortOrder}}
+    @setSortBy={{this.setSortBy}}
+    @setSortOrder={{this.setSortOrder}}
+    @columns={{array "artist" "album" "release year"}}
```

Comment:
I think if they want sortable columns, then it's whatever they can pass as an array. This was the most basic simple example on purpose. We can add more complex use cases as examples once we've worked out some of the remaining issues.

Reply:
From what I understand, we're working on the assumption that the column "titles" and the model "keys" are exactly the same (we call `sort-by` on the model using the `column` string that is used to populate the table header cells. I don't think this is always the case, so we should consider this because it may lead to a `TableSortable` that can't be used properly in Cloud UI.

/cc @alex-ju

Reply:
re: icons in header cells- design has confirmed that we're not permitting icons (other than the sorting icons which happen automatically) in header cells. so I don't think we have to worry about that right now.

Reply:
I'll do some more examples to show that it doesn't just have to be a string, but didn't include them initially because the Ember documentation shows this sort of thing already. I'll add it just to be safe.